ansible/roles/pretix/templates/pretix-maintenance.sh.j2

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+##
+## This script checks the current UTC time, and based on that time
```

Comment:
Perhaps we should mention why we do this (see my issue)

Reply:
You're right!